vowpalwabbit/CPackConfig.cmake

```diff
@@ -0,0 +1,88 @@
+# This file will be configured to contain variables for CPack. These variables
```

Comment:
Did you intend to include this file in the PR? If so, consider removing properties that are specific to your environment, such as `set(CPACK_BUILD_SOURCE_DIRS "/mnt/c/Users/v-zmhammedi/vowpal_wabbit;/mnt/c/Users/v-zmhammedi/vowpal_wabbit/vowpalwabbit")`, or consider abstracting them out based on other CMake variables/properties.